processor/src/main/java/org/infinispan/protostream/annotations/impl/processor/AutoProtoSchemaBuilderAnnotationProcessor.java

```diff
@@ -59,10 +59,12 @@
 public final class AutoProtoSchemaBuilderAnnotationProcessor extends AbstractProcessor {
 
    /**
-    * The only option we support: activate debug logging.
+    * Activate debug logging.
     */
    public static final String DEBUG_OPTION = "debug";
 
+   public static final String FULLY_QUALIFIED_ANNOTATIONS = "fully-qualified-annotations";
```

Comment:
maybe write some comment where we can enable this and what it does.

Reply:
I have added some info in `README.md`